drivers/soundwire/intel_shim2.c

```diff
@@ -158,17 +158,52 @@ static void intel_debugfs_exit(struct sdw_intel *sdw) {}
 
 static int intel_link_power_up(struct sdw_intel *sdw)
 {
+	unsigned int link_id = sdw->instance;
+	struct sdw_bus *bus = &sdw->cdns.bus;
+	u32 *shim_mask = sdw->link_res->shim_mask;
+	struct sdw_master_prop *prop = &bus->prop;
+	u32 syncprd;
```

Comment:
nit-pick: can we follow our coding style of structs first and then the others?

Reply:
sure.

Reply:
done